blocks/filter.py

```diff
@@ -96,6 +96,16 @@ class object contained in the list.
     """
     def __init__(self, roles=None, bricks=None, each_role=False, name=None,
                  name_regex=None, applications=None):
+        if bricks is not None and not isinstance(bricks, Brick):
```

Comment:
Sorry I forgot to write that I updated it right away, it should be ok right now. :)